__tests__/topics.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require("supertest");
+const app = require("../app");
+const pool = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+const testData = require("../db/data/test-data");
+
+afterAll(() => {
+  if (pool.end) pool.end();
+});
+
+beforeAll(() => {
+  return seed(testData);
+});
+
+describe("Topics api testing", () => {
+    
+  test("GET the status of /api/topics", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+    
+  test("GET the response and status of /api/topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("get exact response from /api/topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+
+        expect(topics).toBeInstanceOf(Array);
```

Comment:
oh wait you seem to have done this here, I might have done them in the same test but good work 👍